### PR TITLE
Chromium 66

### PR DESCRIPTION
**Pull request by alexeykuzmin**
Project board: https://github.com/electron/electron/projects/10
Libcc PR: https://github.com/electron/libchromiumcontent/pull/499

### CODE REVIEW

**Comment by ibash** [comment]
Is it done yet?

**Comment by Ontopic** [comment]
Ahhh, almost there. Thank you guys for your hard work so far.

Not sure if it's helpful, but with previous builds of my local Chromium checkout I saw similar errors: https://chromium-review.googlesource.com/c/chromium/src/+/941901

**Comment by skreborn** [comment]
@alexeykuzmin Is there any ETA on this?

_A production application I'm working on using Electron really needs support for the Unicode Property Escapes feature that's not supported by version 61, but is available for use in version 66._

**Comment by ibash** [comment]
Iirc they don’t promise dates!

Islam Sharabash
217-377-9657

Sent via Superhuman iOS ( https://sprh.mn/?vip=islam.sharabash@gmail.com )

**Comment by alexeykuzmin** [comment]
@skreborn We are working on this.
We have an estimate internally but don't want to share it because it would look like a promise, and we don't want to make any. @ibash is right =)

**Comment by skreborn** [comment]
@alexeykuzmin Perfectly understandable. When I said "estimate", I really meant an estimate though, even if that's just "anywhere in the next 3 months" - but that's okay. I know that these kinds of things tend to have a backlash when they just happen to be inaccurate for some reason, so I'll just wait patiently and make-do with the current version until this is ready. Thanks for the reply!

**Comment by Ontopic** [comment]
Party time seems close, it's working on my mac

File: atom/browser/api/atom_api_desktop_capturer.cc [hunk 1]
```diff
@@ -58,17 +58,11 @@ DesktopCapturer::~DesktopCapturer() {}
 void DesktopCapturer::StartHandling(bool capture_window,
                                     bool capture_screen,
                                     const gfx::Size& thumbnail_size) {
-  webrtc::DesktopCaptureOptions options =
-      content::desktop_capture::CreateDesktopCaptureOptions();
-#if defined(OS_WIN)
-  using_directx_capturer_ = options.allow_directx_capturer();
```
Comment (alexeykuzmin) [inline atom/browser/api/atom_api_desktop_capturer.cc]: @nitsakh Are you sure this should be removed?

File: brightray/brightray.gyp [hunk 2]
```diff
@@ -199,7 +199,6 @@
                 'libraries': [
                   # Following libraries are always linked statically.
                   '<(libchromiumcontent_dir)/libhttp_server.a',
-                  '<(libchromiumcontent_dir)/libdesktop_capture.a',
```
Comment (alexeykuzmin) [inline brightray/brightray.gyp]: @nitsakh 
Then we shouldn't build it in libcc
https://github.com/electron/libchromiumcontent/blob/chromium-upgrade/66/chromiumcontent/BUILD.gn#L98

File: atom/browser/api/atom_api_desktop_capturer.cc (hunk 1 above)
Comment (nitsakh) [reply atom/browser/api/atom_api_desktop_capturer.cc]: Yeah, chromium does that. https://cs.chromium.org/chromium/src/content/public/browser/desktop_capture.cc?rcl=92e0446435896425805d05af8a91072263223d68&l=14

File: brightray/brightray.gyp (hunk 2 above)
Comment (nitsakh) [reply brightray/brightray.gyp]: Right, I am making that change.

File: atom/browser/api/atom_api_desktop_capturer.cc (hunk 1 above)
Comment (alexeykuzmin) [reply atom/browser/api/atom_api_desktop_capturer.cc]: But we use `using_directx_capturer_ ` here
https://github.com/electron/electron/blob/bc2387cf1c02182c47771a65d69c1a6ff946a172/atom/browser/api/atom_api_desktop_capturer.cc#L97

**Comment by NAZMULBABLU** [comment]
Ok

**Comment by miniak** [comment]
@alexeykuzmin, Chromium 65 dropped support for OS X 10.9. The deployment target should therefore be bumped to 10.10. And also it should be documented.

File: atom/browser/atom_browser_client.cc [hunk 3]
```diff
@@ -472,6 +481,33 @@ void AtomBrowserClient::SiteInstanceDeleting(
   }
 }
 
+std::unique_ptr<net::ClientCertStore> AtomBrowserClient::CreateClientCertStore(
```
Comment (miniak) [inline atom/browser/atom_browser_client.cc]: @alexeykuzmin please use the changes from
https://github.com/electron/electron/pull/13245/files#diff-5a6becb7577816fc1d0a6ac6dc66db11

**Comment by alexeykuzmin** [comment]
@electron/docs , @electron/notifications , @electron/printing 
Can you please review and approve the PR?

**Comment by JeganMSD** [comment]
@MarshallOfSound what's reason to remove this event (did-get-response-details) from WebView??

**Comment by Trevorulliac** [comment]
Merge

**Pull request merged**